hms-app/src/pages/Archive.tsx

```diff
@@ -24,10 +24,12 @@ export default function Archive() {
         </>
       )}
 
-      <HackathonDetails
-        hackathonId={selectedHackathonId}
-        type={HackathonDetailsType.Archive}
-      />
+      {selectedHackathonId !== '' && (
```

Comment:
I think that just this works...
```suggestion
      {selectedHackathonId && (
```